pkg/gui/view_helpers.go

```diff
@@ -62,8 +62,16 @@ func (gui *Gui) resizeCurrentPopupPanel() error {
 }
 
 func (gui *Gui) resizePopupPanel(v *gocui.View, content string) error {
-	x0, y0, x1, y1 := gui.getConfirmationPanelDimensions(v.Wrap, content)
-	_, err := gui.g.SetView(v.Name(), x0, y0, x1, y1, 0)
+	x0, y0, _, y1 := v.Dimensions()
```

Comment:
There's currently an issue with the resizing when we have the setup of commit message panel and description below it. When we cycle through commits the resizing of the description goes wrong because it can't seem to calculate the correct height (see video).

This workaround works for this particular usecase because with the new setup we don't have to resize the height vertically (still resizes horizontally though). I don't know if this will be an issue with other popups though, what do you suggest we do here? 


https://user-images.githubusercontent.com/30256079/218268410-49abdeb3-f4e0-4adc-8d77-b024ade9f2b9.mov

Reply:
We can't rely on the existing function because we need to resize the message and description view independently, but I still would like us to have code for making the commit description view taller if there's more content there. I would have a separate function specifically for resizing both the message and description views. We can also say within `resizeCurrentPopupPanel` that if the current view is the message or description view then we call our new resize method